xml/obs/devel:LEO.xml

```diff
@@ -0,0 +1,18 @@
+<openQA
+    project_pattern="devel:LEO"
+    dist_path="images"
+    distri="alp"
+    archs="x86_64">
```

Comment:
I know that @ggardet is planning to work on arm images, maybe we could also include `aarch64` here already? Will the fail if it can't find those images?

Reply:
hm... good point, I will try to make it work for existing images and then add `aarch64` so everything is prepared ahead

Reply:
Yes, please include aarch64 asap please :)